### ISSUE TITLE
error TypeError: window.NativeShell.getPlugins

### ISSUE THREAD

**Issue by douglasppaz** [jellyfin/jellyfin-android#149]
App opens but just have a black screen.
I get the following errors in console (I'm using chrome://inspect to access this log).

Git branch: master
Java JDK: installed 1.8.0
Android target: installed android-28

```
Refused to load the image 'android-webview-video-poster:default_video_poster/-4318720441476959411' because it violates the following Content Security Policy directive: "default-src * 'self' 'unsafe-inline' 'unsafe-eval' data: gap: file: filesystem: ws: wss:". Note that 'img-src' was not explicitly set, so 'default-src' is used as a fallback.
```

```
site.js:1 Uncaught (in promise) TypeError: window.NativeShell.getPlugins is not a function
    at site.js:1
    at site.js:1
    at Object.execCb (alameda.js:1)
    at a (alameda.js:1)
    at Object.u [as depFinished] (alameda.js:1)
    at alameda.js:1
```

**Comment by dkanada** [comment]
That was my bad, forgot to merge the Android changes after a pull request in web. This will be fixed soon. We actually have an official build on Google Play now if you are just looking for an installation.

**Comment by douglasppaz** [comment]
Great! 
Thanks for the answer.

Notify me as soon as I fix the problem.  I want to contribute to the project as well.

**Comment by dkanada** [comment]
will be fixed with #150

**Comment by dkanada** [comment]
@douglasppaz this has been fixed and you should be able to build the app now.
